src/libcommon/utility/types.h

```diff
@@ -332,19 +343,41 @@ enum class AppStateKey {
     LogUploadState,
     LogUploadPercent,
     LogUploadToken,
+    AppUid,
     Unknown  //!\ keep in last position (For tests) /!\\ Only for initialization purpose
 };
+
 constexpr int64_t SELF_RESTARTE_DISABLE_VALUE = -1;
 constexpr int64_t SELF_RESTARTER_NO_CRASH_DETECTED = 0;
 
 enum class LogUploadState { None, Archiving, Uploading, Success, Failed, CancelRequested, Canceled };
-
 enum class UpdateState { Error, None, Checking, Downloading, Ready, ManualOnly, Skipped };
 
+enum class UpdateStateV2 { UpToDate, Available, Downloading, Ready, Error };
+enum class DistributionChannel { Prod, Next, Beta, Internal, Unknown };
+enum class Platform { MacOS, Windows, LinuxAMD, LinuxARM, Unknown };
+
+struct VersionInfo {
+        std::string tag;
```

Comment:
The `tag` member variable should be documented in. a docstring: Is it a `git` tag?